js/home/index.js

```diff
@@ -10,3 +10,19 @@ function renderCard() {
 }
 
 renderCard();
+
+// for (let node of document.querySelectorAll(".card__bookmark")) {
```

Comment:
I'd rather not have that piece of uncommented be merged into the mainline branch